core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,1211 @@
+      <h2 class="oppia-about-learn-title-mobile" [innerHTML]="'I18N_ABOUT_PAGE_LEARN_FROM' | translate"></h2>
+      <div class="oppia-about-section-four-right">
+        <picture class="oppia-about-lesson-image">
+          <source type="image/webp" [srcset]="getStaticImageUrl('/about/fraction_lesson.webp')">
+          <source type="image/png" [srcset]="getStaticImageUrl('/about/fraction_lesson.png')">
+          <img [src]="getStaticImageUrl('/about/fraction_lesson.png')" class="oppia-about-lesson-image" alt="lesson image">
+        </picture>
+      </div>
+      <div class="oppia-about-section-four-left">
+        <h2 class="oppia-about-learn-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_LEARN_FROM' | translate"></h2>
+        <p class="oppia-about-learn-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_LEARN_FROM_CONTENT' | translate"></p>
+        <div class="oppia-about-classroom-button-container">
+          <form action="<[$ctrl.classroomUrl]>">
+            <button type="submit" (click)="onClickVisitClassroomButton()"
+                    class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
+                    [innerHTML]="'I18N_ACTION_VISIT_CLASSROOM' | translate">
+            </button>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-about-page-section-five">
+    <div class="oppia-section-five">
+      <h2 class="oppia-about-explore-lesson-title-mobile" [innerHTML]="'I18N_ABOUT_PAGE_EXPLORE_LESSONS' | translate"></h2>
+      <div class="oppia-about-section-five-right">
+        <picture class="oppia-about-library-image">
+          <source type="image/webp" [srcset]="getStaticImageUrl('/about/library_layout.webp')">
+          <source type="image/png" [srcset]="getStaticImageUrl('/about/library_layout.png')">
+          <img [src]="getStaticImageUrl('/about/library_layout.png')" alt="library image">
+        </picture>
+      </div>
+      <div class="oppia-about-section-five-left">
+        <h2 class="oppia-about-explore-lesson-title oppia-about-page-title" [innerHTML]="'I18N_ABOUT_PAGE_EXPLORE_LESSONS' | translate"></h2>
+        <p class="oppia-about-explore-lesson-content oppia-about-page-content" [innerHTML]="'I18N_ABOUT_PAGE_EXPLORE_LESSONS_CONTENT' | translate"></p>
+        <div class="oppia-about-classroom-button-container">
+          <form action="<[$ctrl.classroomUrl]>">
+            <button type="submit" (click)="onClickBrowseLibraryButton()"
+                    class="btn oppia-about-button oppia-about-button-browse oppia-about-browse-button-position oppia-transition-200"
+                    [innerHTML]="'I18N_ACTION_BROWSE_LIBRARY' | translate">
+            </button>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="oppia-about-page-section-six" ng-style="{'background-image': '<[$ctrl.getCommunityBackgroundUrl()]>'}">
```

Comment:
Ditto! Can you please please go through this file and work on cleaning and restricting it better?

Reply:
removed